drgn/helpers/linux/__init__.py

```diff
@@ -33,9 +33,9 @@
 import importlib
 import pkgutil
 
-__all__ = []
+__all__ = []  # type: ignore
```

Comment:
Even though it's somewhat magic, we might as well annotate it with its actual type.

```suggestion
__all__: List[str] = []
```

Reply:
For some reason my brain thought it was a `Dict[str, Any]` and that it wouldn't be worth annotating. Will use the List[str] instead.